swap_meet/item.py

```diff
@@ -1,2 +1,23 @@
+
 class Item:
-    pass
\ No newline at end of file
+    category = ""
+    
+    def __init__(self, category = category, condition = 0):
+        self.category = category
+        self.condition = condition
```

Comment:
We don't need to use a class variable in the default parameter values here. Strings are immutable, and we can list a string value as a default variable just like we can the integer value for `condition`.

With respect to formatting, keep in mind that [PEP8](https://peps.python.org/pep-0008/#other-recommendations) recommends no spaces around `=` when used for default values or keyword arguments. So we might prefer to write

```python
    def __init__(self, category="", condition=0):
        self.category = category
        self.condition = condition
```